sdk/communication/AzureCommunicationChat/Source/Generated/ChatThreadOperation.swift

```diff
@@ -852,58 +705,47 @@ public final class AzureCommunicationChatService {
         withOptions options: UpdateChatMessageOptions? = nil,
         completionHandler: @escaping HTTPResultHandler<Void>
     ) {
-        // Construct URL
-        let urlTemplate = "/chat/threads/{chatThreadId}/messages/{chatMessageId}"
-        let pathParams = [
-            "chatThreadId": chatThreadId,
-            "chatMessageId": chatMessageId,
-            "$endpoint": client.endpoint.absoluteString
-        ]
-        // Construct query
-        let queryParams: [QueryParameter] = [
-            ("api-version", "2020-09-21-preview2")
-        ]
+        // Create request parameters
+        let params = RequestParameters(
+            (.path, "chatThreadId", chatThreadId, .encode), (.path, "chatMessageId", chatMessageId, .encode),
+            (.uri, "endpoint", client.endpoint.absoluteString, .skipEncoding),
+            (.query, "api-version", "2020-11-01-preview3", .encode),
+            (.header, "Content-Type", "application/merge-patch+json", .encode),
+            (.header, "Accept", "application/json", .encode)
+        )
 
-        // Construct headers
-        var headers = HTTPHeaders()
-        headers["Content-Type"] = "application/json"
-        headers["Accept"] = "application/json"
         // Construct request
```

Comment:
For both update methods removed usage of MergePatch(), also changed self.options.logger to client.options.logger 

https://github.com/knvsl/azure-sdk-for-ios/blob/ef29c1fe9edaa18848d01f4db7044904731bf294/sdk/communication/AzureCommunicationChat/Source/Generated/ChatThreadOperation.swift#L731

Reply:
These should be opened as issues in autorest, not manually changed in the generated code.